### ISSUE TITLE
Cannot Add Entities to Device Once Added

### ISSUE THREAD

**Issue by MosheTzvi** [rospogrigio/localtuya#1136]
## The problem
After adding a new Device via the UI and initially adding some entities, there is no way to go back to that device and add additional entities for it.  I experienced this with a thermostat device that had multiple values I wanted to be able to control/track in HA.  There is no way to add any more entities for that device after initial setup and there is no way to delete that device so I can start over without deleting localTuya completely.

## Environment

- Localtuya version: V4.1.1
- Last working localtuya version (if known and relevant): 
- Home Assistant Core version: 
- Home Assistant 2022.11.3
Supervisor 2022.10.2
Operating System 9.3
Frontend 20221108.0 - latest
- [x] Are you using the Home Assistant Tuya Cloud component ? 
- [x] Are you using the Tuya App in parallel ? 

## Steps to reproduce

1. Add a new Device
2. Add some entities
3. Finish the initial setup
4. Try and add another entity for that device

## Configuration `configuration.yaml` or `config_flow`

```yaml

```

## Provide Home Assistant taceback/logs

```txt

```

**Comment by meracing** [comment]
this would be great if possible. I have some plugs so entities are often with value 0. means it is hard to figure out what the entity is. and if you do one wrong you have to delete the localtuya and start all over again. if it would only be the one device it would help a lot.

**Comment by Froh987** [comment]
I ran in to this issue as well.  It actually took me awhile to figure how how to just add multiple entities to the device by making sure the "Do not add any more entities " check box was unchecked.  However, as stated above, it seems impossible to add more entities through the edit menu, once the device has been finalized.

That said, I did find that it was possible to just delete the device without having to delete the whole localtuya integration.  To do so, just click the device, then click the three vertical dots under the "Device Info" section, and then click "Delete."  That should get rid of just that device, and the remaining localtuya devices will still be there.

**Comment by MosheTzvi** [comment]
> I ran in to this issue as well. It actually took me awhile to figure how how to just add multiple entities to the device by making sure the "Do not add any more entities " check box was unchecked. However, as stated above, it seems impossible to add more entities through the edit menu, once the device has been finalized.
> 
> That said, I did find that it was possible to just delete the device without having to delete the whole localtuya integration. To do so, just click the device, then click the three vertical dots under the "Device Info" section, and then click "Delete." That should get rid of just that device, and the remaining localtuya devices will still be there.

Only problem with that is that any automations dealing with entities from that device need to be redone.  Really needs to be an option to add more entities from a device.

**Comment by rfam13** [comment]
This started in version 4.0, as I remember being able to add more entities (dps) in the prior version before cloud support, I have this same issue posted from 6 months ago when it was released with no resolution unfortunately.

I honestly thought it was just a small oversight that would be fixed once it was noticed.

**Comment by Laho812** [comment]
I have the same issue. I cannot add entities by editing a device. In the meantime i set the "Do not add any more entities " check box to unchecked by default by editing config_flow.py

**Comment by rfam13** [comment]
> I have the same issue. I cannot add entities by editing a device. In the meantime i set the "Do not add any more entities " check box to unchecked by default by editing config_flow.py
> 

This allows you to add more when editing a device?  Do you just cancel out one finished adding?

**Comment by Laho812** [comment]
> > I have the same issue. I cannot add entities by editing a device. In the meantime i set the "Do not add any more entities " check box to unchecked by default by editing config_flow.py
> 
> This allows you to add more when editing a device? Do you just cancel out one finished adding?

No, but i do not accidentally finish the process too early. There's no chance to forget unchecking the check box.

**Comment by rfam13** [comment]
Nice tip, thanks!

**Comment by qisbic** [comment]
It is unfortunate.. such a basic feature has not been fixed for a long time..  as a developer.. one rule I have it.. I eat my own dog food.. if it broke it.. I fix it.

**Comment by adizag** [comment]
same here
that should be a basic feature.
Thanks in advance to the developer!
